Sample.java

```diff
@@ -1,7 +1,110 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Prob-1: Reverse LL
+// Time Complexity :O()

```

Comment:
Add it please

Reply:
Time Complexity = O(n) where n is number of nodes to reverse in the LL
Space Complexity = O(n) where n is number of nodes to reverse in the LL